thriftbp/client_pool.go

```diff
@@ -211,6 +211,11 @@ type ClientPoolConfig struct {
 	//
 	// Optional. If this is empty, no "User-Agent" header will be sent.
 	ClientName string `yaml:"clientName"`
+
+	// The hostname to add as a "thrift-hostname" header.
+	//
+	// Optional. If empty, not "thrift-hostname" header will be sent.
+	ThriftHostnameHeader string `yaml:"thriftHostnameHeader"`
```

Comment:
```suggestion
	// The hostname to add as a "Thrift-Hostname" header.
	//
	// Optional. If empty, no "Thrift-Hostname" header will be sent.
	ThriftHostnameHeader string `yaml:"thriftHostnameHeader"`
```

Reply:
well from the rest of the convo it looks like we'll keep the lower case in the end

Reply:
but we still should `s/not/no/` :)